src/Pages/Main/Main.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {
+    Button, Checkbox, List, ListItem, ListItemText, Divider,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import { DatePicker } from 'antd';
+import Loader from '../../Components/Loader/Loader';
+import Avatar from '../../Components/Avatar/Avatar';
+import { DojotButton, CleanButton, CustomButton } from '../../Components/Button';
+import 'antd/dist/antd.css';
+
+
+function Checkboxes() {
+    return (
+        <>
```

Comment:
<> ? Is this usual?

Reply:
Yes. Is the short notation for Fragment tag.